homeassistant/components/smartthings/smartapp.py

```diff
@@ -405,7 +405,7 @@ async def _continue_flow(
         (
             flow
             for flow in hass.config_entries.flow.async_progress_by_handler(DOMAIN)
-            if flow["context"]["unique_id"] == unique_id
+            if flow["context"].get("unique_id") == unique_id
```

Comment:
Suggestion:
```python
if flow["handler"] == DOMAIN and flow["context"].get("unique_id") == unique_id```
```
May be you can test this.

See: https://github.com/home-assistant/core/pull/38667#discussion_r467525459

Reply:
The domain check has been removed since, why does it need to be there? The only thing that is failing is that if the dictionary element isn't there, the code crashes.

Reply:
Okay, fine